src/pages/oversikt/kandidatliste/Kandidatliste.tsx

```diff
@@ -16,13 +16,19 @@ interface Props {
 
 const Kandidatliste = ({ kandidater, onClickKandidat, antallValgteKriterier }: Props) => {
     const [visKandidater, toggleVisKandidater] = useState<boolean>(true);
+    const [counter, setCounter] = useState<number>(0);
```

Comment:
Mulig å finne et mer forklarende navn her?
Eventuelt endre koden for å gjøre det mer tydelig hva som skal skje